tests/ui/target-feature/abi-required-target-feature-flag-disable.loongarch.stderr

```diff
@@ -0,0 +1,13 @@
+warning: target feature `d` must be enabled to ensure that the ABI of the current target can be implemented correctly
+   |
+   = note: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
+   = note: for more information, see issue #116344 <https://github.com/rust-lang/rust/issues/116344>
+
+warning: unstable feature specified for `-Ctarget-feature`: `d`
+   |
+   = note: this feature is not stably supported; its behavior can change in the future
+
+warning: both target-abi and the triple-implied ABI are invalid, ignoring and using feature-implied ABI
```

Comment:
@heiher any idea what this warning is about? Does it come from LLVM?

Reply:
Yes, they are quite similar. The LoongArch backend attempts to compute the ABI based on the `target-abi` and `triple-name`. If there’s a conflict between the resulting ABI and the target features, the final ABI will be determined by the target features.

Fore more details:

https://github.com/rust-lang/llvm-project/blob/rustc-1.86.0/llvm/lib/Target/LoongArch/MCTargetDesc/LoongArchBaseInfo.cpp#L171-L172

Reply:
Okay. I find the LoongArch very unclear compared to the RISC-V error... but anyway it should be impossible to trigger this warning from rustc without also triggering "target feature `d` must be enabled to ensure that the ABI of the current target can be implemented correctly" so users will hopefully understand what the problem is.